docs/installation/certificate.asciidoc

```diff
@@ -0,0 +1,319 @@
+
+- Go to Configuration > System Configuration > SSL Certificates
+
+image::certificate/14-HTTP-CSR.png[scaledwidth="100%",alt="CSR"]
+
+- Click on “Generate Signing Request (CSR)
+
+- Complete the following using your own information, Common Name must match the captive portal hostname in Configuration > General Configuration > Hostname
+
+image::certificate/16-CSR.png[scaledwidth="100%",alt="CSR"]
+
+Warning: This capture have been made on PacketFence V13. If you are using a lower version (not under V11.2) Subject Alternative Names will be automatically generated from Common Name field.
+
+- Save the CSR and private key to a secure location, you will need it to renew your certificate.
+
+- You can also generate the CSR with CLI using:
+
+[source, shell]
+----
+openssl req -newkey rsa:2048 -keyout PRIVATEKEY.key -out MYCSR.csr
+----
+
+==== Submit the CSR to a Certificate Authority (CA)
+
+Once you have generated the CSR, the next step is to submit it to a Certificate Authority (CA) for validation and issuance of the SSL/TLS certificate. There are many CAs to choose from, and it is important to select a reputable one that is trusted by major web browsers.
+
+To submit the CSR to a CA, follow these steps:
+
+- Select a CA and follow their instructions for submitting a CSR.
+
+- The Subject Alternative Name must exactly match the captive portal hostname in Configuration > General Configuration > Hostname.
+
+- Ensure that your CA supports X509 in base 64 format.
+
+- Provide the CSR and any other required information, such as payment and proof of identity.
+
+- Wait for the CA to validate the CSR and issue the SSL/TLS certificate.
+
+- Download the CA certificate in Apache format.
+
+If your CA provides the certificate in PKCS12 format, you can use the following commands to extract the .crt and .key files:
+
+[source, shell]
+----
+openssl pkcs12 -in {{.temp_dir}}/{{.input.cn}}.p12 -clcerts -nokeys -out {{.temp_dir}}/{{.input.cn}}.crt -passin pass:secret
+openssl pkcs12 -in {{.temp_dir}}/{{.input.cn}}.p12 -nocerts -nodes -out {{.temp_dir}}/{{.input.cn}}.key -passin pass:secret
+----
+
+==== Install the SSL/TLS Certificate on the Server
+
+Once you have received the SSL/TLS certificate from the CA, the final step is to install it on PacketFence. This involves configuring the web server to use the SSL/TLS certificate for encrypted communications.
+
+To install the SSL/TLS certificate, follow these steps:
+
+- Open the web admin interface.
+
+- Go to `Configuration > System Configuration > SSL Certificates > HTTP > Edit HTTP Certificates`.
+
+image::certificate/1-HTTP.png[scaledwidth="100%",alt="HTTP"]
+
+- Import or open your certificate file (.crt) with a text editor and copy/paste the content into the "HTTPs Server Certificate" field.
+
+image::certificate/2-HTTP-Certificate.png[scaledwidth="100%",alt="Certificate"]
+
+- Import or open your private key file (.key) and copy/paste the content into the "HTTP Server Private Key" field.
```

Comment:
If the CSR has been generated on PF server (from webadmin), you need to keep existing key in place (because key has been used to sign CSR).

Reply:
Sorry I don't get your point.